biothings/hub/__init__.py

```diff
@@ -124,7 +124,7 @@ def _config_for_app(config_mod=None):
 
 
 def get_loop(max_workers=None):
-    loop = asyncio.get_event_loop()
+    loop = asyncio.get_event_loop_policy().get_event_loop()
```

Comment:
@sengineer0 can you please explain this change?

Reply:
As I understand, `asyncio.get_event_loop()` will raise a warning if there is no current event loop (from python 3.11, it will raise a `RuntimeError` instead)

So I use `asyncio.get_event_loop_policy().get_event_loop()` to use the main thread for the loop.
This idea come from a commit in the pytest-asyncio
https://github.com/pytest-dev/pytest-asyncio/pull/214/files#diff-dd38507a48269262cbe9feadbc8d8f4f5caefc407d6fed73e3e404a2f537678aR15

Actually, even I tested again the Hub flow, but I still not 100% sure about this change.
If you have other idea, feel free to share with me :)
@newgene

Reply:
The recent release notes of tornado v6.2 look like mentioned something similar, see if you can learn how they fixed this issue:

https://www.tornadoweb.org/en/stable/releases/v6.2.0.html

Reply:
sure, I will check it for information :)

Reply:
@newgene 
The recent release notes of tornado said that `The new recommended pattern is to start the event loop with asyncio.run.`

but as the document of the [async_io](https://docs.python.org/3/library/asyncio-task.html#asyncio.run) noted: `This function always creates a new event loop and closes it at the end. It should be used as a main entry point for asyncio programs, and should ideally only be called once.`

So I think we should not use the tornado's recommendation. Instead, we can keep use `asycnio.get_event_loop`, and add a catch for RuntimeError, when that error occurs, just create a new loop and return.

Reply:
@sengineer0 sounds good.  I would like to see your implementation based on your plan above.

Reply:
never mind, I saw your updated code now. Can you please add some comments to your current implementation?

Reply:
@newgene  sure, i added. 
please have a check: https://github.com/newgene/biothings.api/pull/67/commits/46d79f18e9ab92b1b94ed8245caa456fcb371b96